examples_tests/22.RaytracedAO/common.h

```diff
@@ -8,7 +8,7 @@
 #define MAX_TRIANGLES_IN_BATCH 16384
 
 // need to bump to 2 in case of NEE + MIS, 3 in case of Path Guiding
-#define SAMPLING_STRATEGY_COUNT 1
+#define SAMPLING_STRATEGY_COUNT 2
```

Comment:
good this is what we wanted.

But, in here:
https://github.com/Devsh-Graphics-Programming/Nabla/blob/a16fec199ca775bfc41a0f8e03aff90ff37197bb/examples_tests/22.RaytracedAO/Renderer.cpp#L1155

you want to remove the multiplication by `SAMPLING_STRATEGY_COUNT` cause right now its probably causing 2x the work and mem usage (we don't really send out double the rays, because we resample)

Reply:
Done